assets/src/edit-story/components/fontPicker/pickerContainer.js

```diff
@@ -246,6 +301,12 @@ function FontPickerContainer({ value, onSelect, onClose, isOpen }) {
   const itemRenderer = useCallback(
     ({ service, name }, index) => (
       <>
+        {dividerIndexTracker.current >= 0 && index === 0 && (
+          <Divider>{__('Recently used', 'web-stories')}</Divider>
+        )}
+        {index === dividerIndexTracker.current + 1 && (
+          <Divider>{__('Recommended', 'web-stories')}</Divider>
+        )}
```

Comment:
I get why the headline is part of the item renderer (I designed it that way), but the headline shouldn't be part of the focus rectangle, as that is quite misleading:

![Screen Shot 2020-09-02 at 22 36 41](https://user-images.githubusercontent.com/637548/92065106-f885f180-ed6c-11ea-841a-0a2f1a9ac3d5.png)

Reply:
Yea, the scroll list handles the focus by making the li focusable, but the scroll list only exposes the item renderer (within the li) to alter its contents. So for the same reason the headline is part of the item renderer, it's also part of the focus rectangle.

I didn't see a way to not have this happen without altering the `ScrollList` implementation which I didn't want to mess with for a cosmetic update. Happy to tinker around with it some more to see if I can come up with a solution that doesn't mess with the logic of it too much, but struggling to think of anything.

Reply:
You could remove the focus rectangle from the whole item and apply it to only the list element (when the option has focus), but that requires updating `ScrollList` somehow, but I can't see a clean way of doing that [here](https://github.com/google/web-stories-wp/blob/081e621681da7bc9ddc03a8b736a810b12e44236/assets/src/edit-story/components/fontPicker/scrollList.js#L32-L42).

Let's leave it for now.